app/bundles/CoreBundle/Views/Helper/page_actions.html.php

```diff
@@ -76,6 +78,12 @@
                 echo '  <i class="fa fa-'.$icon.'"></i> ' . $view['translator']->trans('mautic.core.form.' . $action) . "\n";
                 echo "</a>\n";
                 break;
+            case 'close':
+                $icon = 'remove';
+                echo '<a class="'.$btnClass.'" href="' . $view['router']->generate('mautic_' . $routeBase. '_action' ,array("objectAction" => "")) . '" data-toggle="' . $editMode . '"' . $editAttr . $menuLink . ">\n";
```

Comment:
Would have to validate this but I think that `mautic_ROUTEBASE_index` is consistently used as route names for the lists so `->generate('mautic_' . $routeBase. '_index')` might be an option here.